server/StudySharp.ApplicationServices/Commands/AddCourseCommand.cs

```diff
@@ -26,19 +26,22 @@ public AddCourseCommandHandler(StudySharpDbContext sharpDbContext)
 
         public async Task<OperationResult> Handle(AddCourseCommand request, CancellationToken cancellationToken)
```

Comment:
Why did you touch this file?

Reply:
Mb misclick, i dont remember when and why i changed it

Reply:
And i dunno what i changed too